Casks/t/thebrowsercompany-dia@beta.rb

```diff
@@ -0,0 +1,28 @@
+cask "thebrowsercompany-dia@beta" do
+  version "0.32.2"
+  sha256 :no_check
+
+  url "https://releases.diabrowser.com/release/Dia-latest.dmg"
+  name "Dia"
+  desc "A smart browser where you can chat, write, learn, plan and shop with your tabs"
```

Comment:
```suggestion
  desc "Web browser"
```

We try to keep the description as concise as possible (see [documentation](https://docs.brew.sh/Cask-Cookbook#stanza-desc)).